omegaconf/dictconfig.py

```diff
@@ -494,9 +498,9 @@ def setdefault(self, key: Union[str, Enum], default: Any = None) -> Any:
         return ret
 
     def items_ex(
-        self, resolve: bool = True, keys: Optional[List[str]] = None
-    ) -> AbstractSet[Tuple[str, Any]]:
-        items: List[Tuple[str, Any]] = []
+        self, resolve: bool = True, keys: Optional[Sequence[DictKeyType]] = None
+    ) -> AbstractSet[Tuple[DictKeyType, Any]]:
+        items: List[Tuple[DictKeyType, Any]] = []
```

Comment:
Leaving this comment here but it applies to other parts of the code as well:

I see multiple places where the previous code was using only `str` but not `Enum`. Do you know if it's just that the previous code was incorrect, or was there a reason not to have enums? In the latter case, would adding `Enum` back (through `DictKeyType`) be an issue? (I personally have no idea what's the role of Enums here...)

Reply:
Enum keys are were added in OmegaConf 2.0 and I am not surprised there are some annotation gaps for it.

Reply:
> Enum keys are were added in OmegaConf 2.0 and I am not surprised there are some annotation gaps for it.

Ok thanks, resolving then